src/main/java/dao/AvailabilityTypesDao.java

```diff
@@ -0,0 +1,23 @@
+package dao;
+
+import com.google.inject.Inject;
+import domain.AvailabilityTypes;
+import io.dropwizard.hibernate.AbstractDAO;
+import org.hibernate.SessionFactory;
+
+public class AvailabilityTypesDao extends AbstractDAO<AvailabilityTypes>{
+    @Inject
+    public AvailabilityTypesDao(SessionFactory sessionFactory){
+        super(sessionFactory);
+    }
+
+    public long create(AvailabilityTypes availabilityTypes){
```

Comment:
Why do we need a create method for availability types? Will we ever want to add them programatically instead of using migrations?